src/main/daml/Daml/Finance/Interface/Account/Account.daml

```diff
@@ -22,13 +23,24 @@ type R = Reference
 -- | Type synonym for `View`.
 type V = View
 
--- | Controllers of the account (related to transfers).
+-- | Frozen details.
+data Frozen = Frozen
+  with
+    unfreezers : Parties
+      -- ^ Parties who have frozen the account and are authorized to unfreeze it.
+    context : Text
+      -- ^ Why the `Account` was frozen.
+  deriving (Eq, Show)
+
+-- | Controllers of the account.
 data Controllers = Controllers
   with
     outgoing : Parties
-      -- ^ Parties instructing a transfer (outgoing).
+      -- ^ Parties instructing an outgoing transfer of a `Holding`.
     incoming : Parties
-      -- ^ Parties approving a transfer (incoming).
+      -- ^ Parties approving an incoming transfer of a `Holding`.
+    freezers : Parties
```

Comment:
I would be more specific in the comment: all parties must authorize or any party must authorize?

Reply:
I changed that.